source/_integrations/imap_email_content.markdown

```diff
@@ -1,4 +1,4 @@
----
+git@github.com:smcpeck/home-assistant.io.git---
```

Comment:
```suggestion
---
```

Not sure if this would somehow disappear automatically or if it needs to be removed now.